test/elementhandle.spec.js

```diff
@@ -52,6 +52,28 @@ module.exports.addTests = function({testRunner, expect}) {
     });
   });
 
+  describe('ElementHandle.boxModel', function() {
+    it('should work', async({page, server}) => {
+      await page.setViewport({width: 500, height: 500});
+      await page.setContent('<div style="width: 100px; height: 100px; margin: 5px; padding: 4px; border: 1px solid black;top: 0; left: 0; position: absolute;"></div>');
+      const elementHandle = await page.$('div');
+      const box = await elementHandle.boxModel();
+
+      expect(box.content).toEqual([10, 10, 110, 10, 110, 110, 10, 110]);
```

Comment:
In what coordinates are the returned numbers? Can we have a test with nested frames to make the coordinate system clear?

Reply:
done, 10x!